lib/devise/models/authenticatable.rb

```diff
@@ -62,7 +62,7 @@ module Authenticatable
         :remember_token, :unconfirmed_email, :failed_attempts, :unlock_token, :locked_at]
 
       include Devise::DeprecatedConstantAccessor
-      deprecate_constant "BLACKLIST_FOR_SERIALIZATION", "Devise::Models::Authenticatable::UNSAFE_ATTRIBUTES_FOR_SERIALIZATION"
+      deprecate_constant "BLACKLIST_FOR_SERIALIZATION", "Devise::Models::Authenticatable::UNSAFE_ATTRIBUTES_FOR_SERIALIZATION", deprecator: Devise.deprecator
```

Comment:
I believe we need to pass it explicitly because we're using [Rails' version of `deprecate_constant`](https://api.rubyonrails.org/classes/ActiveSupport/Deprecation/DeprecatedConstantAccessor.html#method-i-deprecate_constant), we have a [copy of that implementation for older versions where it didn't exist](https://github.com/heartcombo/devise/blob/a7d64ae313672488d8f689b3e48dc0601db8528f/lib/devise/rails/deprecated_constant_accessor.rb#L3-L13) and that's where our default is set.

_(in hindsight, it probably shouldn't have that default but require it explicitly too now, but I'm hopeful that code is going away soon anyway)_

Reply:
Oh I see, I understand now. Looking forward to the ✂️ 😁